.env.example

```diff
@@ -11,10 +11,15 @@ APPLICATION_ID=123456789012345678
 ## Usually, env#NODE_ENV is either one of PRODUCTION, DEVELOPMENT, or STAGING.
 NODE_ENV=DEVELOPMENT
 
-# These keys are used for developer mode and debugging.
+# These keys are used for developer mode and debugging. These keys are only used when env#NODE_ENV is set to DEVELOPMENT.
+## Commands and actions will be directly deployed to this guild.
 DEV_GUILD=123456789012345678
+## This snowflake responds to a role that the bot will use as a gatekeeper for development-only commands.
 DEV_UNLOCK_ROLE=123456789012345678
 
+## This token corresponds to a GitHub fine-grained PAT. Preferably, the PAT should belong to a bot account. This is used for certain tools, i.e. release-it.
```

Comment:
include that this isn't required for the docker compose deployment? or do something else about that idk